<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Population Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>250</y>
      <width>697</width>
      <height>225</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Window:</string>
    </property>
    <widget class="QTextEdit" name="opLogs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>671</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>305</width>
      <height>235</height>
     </rect>
    </property>
    <property name="title">
     <string>Input Parameters:</string>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Years to simulate:</string>
       </property>
       <property name="buddy">
        <cstring>inpYearsToSimulate</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inpYearsToSimulate">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Average Fertility Rate:</string>
       </property>
       <property name="buddy">
        <cstring>inpAvgFertilityRate</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inpAvgFertilityRate">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Max Male Fertility Age (yrs):</string>
       </property>
       <property name="buddy">
        <cstring>inpMaxMaleFertilityAge</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inpMaxMaleFertilityAge">
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Max Female Fertility Age (yrs):</string>
       </property>
       <property name="buddy">
        <cstring>inpMaxFemaleFertilityAge</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="inpMaxFemaleFertilityAge">
       <property name="text">
        <string>40</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Min male marriage age (yrs):</string>
       </property>
       <property name="buddy">
        <cstring>inpMinMaleMarriageAge</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="inpMinMaleMarriageAge">
       <property name="text">
        <string>21</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Min female marriage age (yrs):</string>
       </property>
       <property name="buddy">
        <cstring>inpMinFemaleMarriageAge</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="inpMinFemaleMarriageAge">
       <property name="text">
        <string>18</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Avg. life expectancy (yrs):</string>
       </property>
       <property name="buddy">
        <cstring>inpAvgLifeExpectancy</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="inpAvgLifeExpectancy">
       <property name="text">
        <string>75</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>9</y>
      <width>249</width>
      <height>235</height>
     </rect>
    </property>
    <property name="title">
     <string>Statistics:</string>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Elapsed years:</string>
       </property>
       <property name="buddy">
        <cstring>opTotalYears</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="opTotalYears">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Total births:</string>
       </property>
       <property name="buddy">
        <cstring>opTotalBirths</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="opTotalBirths">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Total deaths:</string>
       </property>
       <property name="buddy">
        <cstring>opTotalDeaths</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="opTotalDeaths">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Male population:</string>
       </property>
       <property name="buddy">
        <cstring>opTotalMales</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="opTotalMales">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Female population:</string>
       </property>
       <property name="buddy">
        <cstring>opTotalFemales</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="opTotalFemales">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Sex ratio:</string>
       </property>
       <property name="buddy">
        <cstring>opSexRatio</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="opSexRatio">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Avg male age:</string>
       </property>
       <property name="buddy">
        <cstring>opAvgMaleAge</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="opAvgMaleAge">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Avg female age:</string>
       </property>
       <property name="buddy">
        <cstring>opAvgFemaleAge</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="opAvgFemaleAge">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>9</y>
      <width>131</width>
      <height>139</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnSimulate">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Simulate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Clear Output Window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRestartSimulation">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Restart Simulation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuit">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>481</y>
      <width>691</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Population Detail</string>
    </property>
    <widget class="QTableWidget" name="opPopulationTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>671</width>
       <height>361</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gender</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Age</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Spouse</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Children</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLineEdit" name="opSimulationTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>890</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>inpYearsToSimulate</tabstop>
  <tabstop>inpAvgFertilityRate</tabstop>
  <tabstop>inpMaxMaleFertilityAge</tabstop>
  <tabstop>inpMaxFemaleFertilityAge</tabstop>
  <tabstop>inpMinMaleMarriageAge</tabstop>
  <tabstop>inpMinFemaleMarriageAge</tabstop>
  <tabstop>inpAvgLifeExpectancy</tabstop>
  <tabstop>btnSimulate</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>opTotalBirths</tabstop>
  <tabstop>opTotalDeaths</tabstop>
  <tabstop>opTotalMales</tabstop>
  <tabstop>opTotalFemales</tabstop>
  <tabstop>opAvgMaleAge</tabstop>
  <tabstop>opAvgFemaleAge</tabstop>
  <tabstop>opLogs</tabstop>
  <tabstop>opTotalYears</tabstop>
  <tabstop>opSexRatio</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
